### ISSUE TITLE
Ignoring input in-use in certain cases

### ISSUE THREAD

**Issue by Aethyrion** [Maassoft/ColorControl#256]
This feature ((https://github.com/Maassoft/ColorControl/issues/92) is great, but I would like to be able to define some actions that could be triggered even if the PC's HDMI port is not the one currently in use. For example, I would like to configure a shortcut allowing me to switch the TV to the HDMI port of the PC.

Is this already possible?

Really great app btw, thanks for your work.

**Comment by Maassoft** [comment]
Yes, you can switch to HDMI 1-4 by using the corresponding App of the preset. The "App" dropdown should contain items like 'com.webos.app.hdmi1', which switches the source to the correct HDMI port when the preset is applied.
